.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules/
+cypress.allure.config.ts/
```

Comment:
@karenn32 Why this config has been added here? And the root is wrong, otherwise - the file will be excluded.